frontend/packages/knative-plugin/src/components/pub-sub/form-fields/PubSubSubscriber.tsx

```diff
@@ -22,7 +23,7 @@ const PubSubSubscriber: React.FC = () => {
         setFieldTouched('spec.subscriber.ref.name', true);
         setFieldValue('spec.subscriber.ref.name', selectedValue);
         if (modelResource) {
-          const { apiGroup, apiVersion, kind } = modelResource;
+          const { apiGroup = 'core', apiVersion, kind } = modelResource;
```

Comment:
do we need to default to `core` here in the case of pubSub resources? is it for k8s service??

Reply:
Yes, defaulting it to `core` to support any base K8s resources, in this case yes it is for K8s service.